test/src/test/resources/hadoop-metrics2-accumulo.properties

```diff
@@ -31,10 +31,28 @@
 accumulo.sink.file-all.class=org.apache.hadoop.metrics2.sink.FileSink
 accumulo.sink.file-all.filename=./target/it.all.metrics
 
-accumulo.sink.test-sink.class=org.apache.accumulo.test.functional.util.Metrics2TestSink
-accumulo.sink.test-sink.context=master
-accumulo.sink.test-sink.filename=test.metrics
-accumulo.sink.test-sink.period=7
+#accumulo.sink.http-sink
+# Test sink - will publish to http end-point and optional write to file
+accumulo.sink.http-gc.class=org.apache.accumulo.test.metrics.HttpMetrics2Sink
```

Comment:
Do we still need the Metrics2TestSink class if this one is used by tests instead?

Reply:
No - that class should be removed.